openmetadata-ui/src/main/resources/ui/src/components/Auth/AppAuthenticators/MsalAuthenticator.tsx

```diff
@@ -104,12 +76,35 @@ const MsalAuthenticator = forwardRef<AuthenticatorRef, Props>(
       return user;
     };
 
+    const handleOnLogoutSuccess = () => {
+      for (const key in localStorage) {
+        if (key.includes('-login.windows.net-') || key.startsWith('msal.')) {
+          localStorage.removeItem(key);
+        }
+      }
+      onLogoutSuccess();
+    };
+
+    const login = async () => {
+      try {
+        const response = await instance.loginPopup(msalLoginRequest);
+
+        onLoginSuccess(parseResponse(response));
+      } catch (error) {
+        onLoginFailure(error as AxiosError);
+      }
+    };
+
+    const logout = () => {
+      handleOnLogoutSuccess();
+    };
+
     const fetchIdToken = async (
       shouldFallbackToPopup = false
     ): Promise<OidcUser> => {
       const tokenRequest = {
         account: account || accounts[0], // This is an example - Select account based on your app's requirements
-        scopes: msalLoginRequest.scopes,
+        scopes: ['User.Read'],
```

Comment:
@chirag-madlani , why are we changing this?

Reply:
Was testing refreshToken flow and changed that but it didn't required so remove it. Thanks